frontend/src/app/components/Chrome/Header/Brand/Brand.tsx

```diff
@@ -12,8 +13,8 @@ export default class Brand extends Component {
             fetch("/system/sling/info.sessionInfo.json")
                 .then(response => response.json())
                 .then(responseJSON => {
-                    if (responseJSON['userID'] !== 'anonymous') {
-                        this.setState({ linkText: 'Pantheon | Log Out [' + responseJSON['userID'] + ']' })
+                    if (responseJSON[this.state.id] !== 'anonymous') {
+                        this.setState({ linkText: 'Pantheon | Log Out [' + responseJSON[this.state.id] + ']' })
```

Comment:
So it looks like you've pulled 'userID' out as a constant, but you've pulled it out into the state object. Was that intentional?

Reply:
https://stackoverflow.com/questions/33387090/how-to-rewrite-code-to-avoid-tslint-object-access-via-string-literals

This is a change to handle an tslint rule without disabling it.

Reply:
I changed it to a const in the method. I overlooked your concern before.